### ISSUE TITLE
Unable to load HGFS driver while running under VMware workstation

### ISSUE THREAD

**Issue by andskli** [boxcutter/centos#25]
Running this using vagrant (boxcutter/centos70 from atlas) under VMware workstation I get the following error:

```
Bringing machine 'testvm' up with 'vmware_workstation' provider...
==> testvm: Cloning VMware VM: 'boxcutter/centos70'. This can take some time...
==> testvm: Checking if box 'boxcutter/centos70' is up to date...
==> testvm: Verifying vmnet devices are healthy...
==> testvm: Preparing network adapters...
==> testvm: Starting the VMware VM...
==> testvm: Waiting for machine to boot. This may take a few minutes...
    testvm: SSH address: 192.168.33.219:22
    testvm: SSH username: vagrant
    testvm: SSH auth method: private key
    testvm:
    testvm: Vagrant insecure key detected. Vagrant will automatically replace
    testvm: this with a newly generated keypair for better security.
    testvm:
    testvm: Inserting generated public key within guest...
    testvm: Removing insecure key from the guest if its present...
    testvm: Key inserted! Disconnecting and reconnecting using new SSH key...
==> testvm: Machine booted and ready!
==> testvm: Forwarding ports...
    testvm: -- 80 => 8080
    testvm: -- 22 => 2222
==> testvm: Setting hostname...
==> testvm: Configuring network adapters within the VM...
==> testvm: Waiting for HGFS kernel module to load...
The HGFS kernel module was not found on the running virtual machine.
This must be installed for shared folders to work properly. Please
install the VMware tools within the guest and try again. Note that
the VMware tools installation will succeed even if HGFS fails
to properly install. Carefully read the output of the VMware tools
installation to verify the HGFS kernel modules were installed properly.
```

**Comment by ufuchs** [comment]
Hello, I have had the same problem. So please take a look at https://github.com/boxcutter/centos/issues/23
There you will find a workaround.

**Comment by andskli** [comment]
The fix seems to have been implemented in this repo as well, see 92d3c1c. Thanks for that :+1: 
I'm going to make the assumption that this will work once released to atlas.
